mitiq/calibration/calibrator.py

```diff
@@ -96,95 +83,96 @@ def _get_performance(
         performance_symbol = "✔" if mitigation_worked else "✘"
         return performance_symbol, noisy_error, mitigated_error
 
-    def unique_techniques(self) -> Set[MitigationTechnique]:
-        """Returns the unique mitigation techniques used across this
-        collection of experiment results."""
-        return set(strategy.technique for strategy in self.strategies)
-
-    def _technique_results(
-        self, technique: MitigationTechnique
-    ) -> Iterator[Tuple[BenchmarkProblem, Strategy, str, float, float]]:
-        """Yields the results from this collection of experiment results,
-        limited to a specific technique."""
-        for strategy, problem in product(self.strategies, self.problems):
-            if strategy.technique is technique:
-                performance_symbol, nerr, merr = self._get_performance(
+    def log_results(self) -> None:
+        """Prints calibration results in the following form
+        ┌────────────────────────────────────────────┬────────────────────────────────┬─────────────────────────┐
+        │ benchmark                                  │ strategy                       │ performance             │
+        ├────────────────────────────────────────────┼────────────────────────────────┼─────────────────────────┤
+        │ Type: rb                                   │ Technique: ZNE                 │ ✔                       │
+        │ Ideal distribution: {'00': 1.0}            │ Factory: RichardsonFactory     │ Noisy error: 0.1053     │
```

Comment:
I don't think it's super useful to the user to see the "Ideal distribution" so let's drop that here (and below in the cartesian logging).

Reply:
For me it is useful since I can easily see what it should look like in the ideal case. But you are right it could make the output too dirty. Removed.

Reply:
Yeah I can see when it would be helpful, but I'm also worried that the table will become less and less readable if you take larger and large benchmarks (e.g. if you have 4 qubits and many possible outcome states).